src/rust-crypto/RoomEncryptor.ts

```diff
@@ -108,27 +119,21 @@ export class RoomEncryptor {
         const fromServer = await this.room.loadMembersIfNeeded();
         const members = await this.room.getEncryptionTargetMembers();
 
-        if (fromServer) {
+        if (fromServer && !this.lazyLoadedMembersResolved) {
```

Comment:
I'm still not really sure what the return value of `loadMembersIfNeeded` is meant to mean. It seems to be undocumented, and nothing else seems to read it, which makes me doubly cautious.

Is the `fromServer` condition here necessary?

Reply:
I think that if you restart the browser ?

Reply:
Not following you. What happens if you restart the browser?